src/kit/shell/src/shellEngine.c

```diff
@@ -301,7 +301,7 @@ void shellRunCommandOnServer(TAOS *con, char command[]) {
 
   int64_t oresult = atomic_load_64(&result);
 
-  if (regex_match(command, "^\\s*use\\s+[a-zA-Z0-9_]+\\s*;\\s*$", REG_EXTENDED | REG_ICASE)) {
+  if (regex_match(command, "^\\s*use\\s+([a-zA-Z0-9_]+|`.+`)\\s*;\\s*$", REG_EXTENDED | REG_ICASE)) {
```

Comment:
In the case, i.e. use dbname, the dbname might be backwards quoted, e.g. capital letters or Chinese characters of unicode UTF-8.